music_service_async_interface/__init__.py

```diff
@@ -72,6 +72,15 @@ def __lt__(self, other):
 class Session(ABC):
     sess: aiohttp.ClientSession
     obj: Type["Object"]
+    quality: Type[AudioQuality]
+
+    @abstractmethod
+    def __init__(
+        self, obj_cls: Type["Object"], quality_cls: Type[AudioQuality], sess: Optional[aiohttp.ClientSession] = None
```

Comment:
`sess: ... = aiohttp.ClientSession()`

Reply:
No, you can't do this with mutable objects.
Default parameter values are calculated on function definition.
You would create all `Session`s without this argument with the same `aiohttp.ClientSession`s.

Reply:
ok, seems weird to me